docs/src/modules/ROOT/pages/using-timefold-solver/modeling-planning-problems.adoc

```diff
@@ -1465,6 +1464,28 @@ It calls `updateArrivalTime` whenever a `previousCustomer` or `vehicle` changes.
 It then propagates the changes to the subsequent elements,
 stopping when the method results in no change to the variable, or when it reaches the tail.
 
+Another valid approach for defining multiple sources per variable is described as follows:
```

Comment:
I wouldn't mention this; at this moment, there's no reason why anyone would want to do it. If we ever implement the variable on a different entity, then we can write a section to describe that aspect.